save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProjectController.kt

```diff
@@ -515,7 +515,13 @@ class DownloadProjectController(private val configProperties: ConfigProperties,
             discoverAndSaveTestSuites(project, rootTestConfig, testRootPath, gitUrl)
         }
         .flatMap { (rootTestConfig, testSuites) ->
-            initializeTests(testSuites, rootTestConfig, executionId)
+            val testSuiteIds = testSuites.map { it.id!! }.toMutableList()
+            testSuiteIds.sort()
```

Comment:
```suggestion
            val testSuiteIds = testSuites.map { it.id!! }.sorted()
```

Reply:
Since all tests were passed, I didn't think about it. Also the records in DB with ids `ALL`  don't bother me, at least.

Any other reasons to make changeset?

Reply:
About code suggestion - fixed

Reply:
Actually, during the execution, we update record twice, in `upload` function and now in `prepareForGit`, in first case we actually will save `ALL`, but now it's basically the stub, and will be rewritten by actual ids, when we will discover tests.
So 'ALL' could be hold in case of some exceptions between `upload` and  `prepareForGit`

Tests data updated